packages/templates/golang.yaml

```diff
@@ -0,0 +1,25 @@
+{{ define "golang_deps" }}
+{{ if .Values.distribution }}
+{{if eq .Values.distribution "opensuse" }}
+- zypper in -y git {{- if .Values.upx -}} upx {{- end }}
+{{else if eq .Values.distribution "fedora" }}
+- dnf install -y git {{- if .Values.upx -}} upx {{- end }}
+{{else if eq .Values.distribution "ubuntu" }}
+- apt-get install -y git {{- if .Values.upx -}} upx {{- end }}
+{{end}}
+{{end}}
+{{end}}
+
+{{ define "golang_download_packagev" }}
```

Comment:
I really hate this. I need to find a way to make it general so it understands from the context that it needs to put a `v` upfront the version